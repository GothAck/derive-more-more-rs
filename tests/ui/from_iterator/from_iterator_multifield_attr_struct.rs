use derive_more_more::FromIterator;

#[derive(Default, FromIterator)]
#[from_iterator(usize)]
struct Named {
    inner: Vec<usize>,
    test: bool,
}

#[derive(Default, FromIterator)]
#[from_iterator(usize)]
struct Unnamed(Vec<usize>, bool);

fn main() {}
