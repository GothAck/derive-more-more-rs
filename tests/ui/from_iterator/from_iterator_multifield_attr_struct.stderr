error: Struct has more than one field, move the `from_iterator` attribute to the appropriate field
 --> tests/ui/from_iterator/from_iterator_multifield_attr_struct.rs:4:1
  |
4 | #[from_iterator(usize)]
  | ^

error: Struct has more than one field, move the `from_iterator` attribute to the appropriate field
  --> tests/ui/from_iterator/from_iterator_multifield_attr_struct.rs:11:1
   |
11 | #[from_iterator(usize)]
   | ^
